docker-compose.yml

```diff
@@ -44,6 +44,32 @@ services:
     depends_on:
       - mysqldb
 
+  sonarqube:
+    container_name: sonarqube
```

Comment:
A docker-compose container name would be more preferable in my opinion than a fixed name.

Reply:
It was set by mean to link with `sonarscanner`. Now, we can expose one more port for TCP connection. And omit this container_name.